boards/hail/README.md

```diff
@@ -40,3 +111,26 @@ You can also specify the serial port to use:
 ```bash
 make program PORT=/dev/ttyUSB0
 ```
+
+Under Mac OS X, plug in the board and use `ls -ltr /dev/tty*` to find a recently created device:
```

Comment:
The hail boards I received have MAC addresses on the back and were running the blink program.  I believe that it was unnecessary for me to run `make program`.

Probably the text should be updated to state something about this.  Something like:

`The hail board are typically shipped provisioned and with the blink userland program running, so it is not typically necessary to rerun `make program`.  However, if your hail board is not provisioned, or you want to reinstall the kernel, then see the steps below.`

The text could be moved to a higher level file.  I originally was starting with boards/hail/README_MAC.md, but decided that one file was better than two.  I was trying to write it to be useful for the Mac user who just received a Hail and was trying to get started.